assets/js/state/sagas/index.js

```diff
@@ -113,11 +113,14 @@ function* initialDataFetch() {
   yield loadSapSystemsHealthSummary();
 
   const {
-    data: { eula_accepted },
+    data: { eula_accepted, premium_subscription },
   } = yield call(get, '/api/settings');
 
   if (!eula_accepted) {
     yield put(setEulaVisible());
+    if (premium_subscription) {
+      yield put(setIsPremium());
+    }
```

Comment:
I think we can extract this if statement to the upper level :+1:

Reply:
Well, we can, but as it is only used to show eula. I will move it anyway, if we need to use it somewhere else

Reply:
Yeah that was exactly my point :+1: